app/feature/feature-edit-coinsured/src/main/kotlin/com/hedvig/android/feature/editcoinsured/ui/AddCoInsuredBottomSheetContent.kt

```diff
@@ -226,6 +233,37 @@ private fun FetchFromSsnFields(
   }
 }
 
+private fun personNummerInputFormatter(text: AnnotatedString, mask: String): TransformedText {
+  val trimmed = if (text.text.length >= 12) text.text.substring(0..11) else text.text
```

Comment:
One thing I worry about here. If someone does in fact enter more characters. Will they be added to the text field but simply not show in the UI? Like if they add 12, then press 10 more characters, will all those still be in the state? Will they then have to also delete 10 times to get back to the original state?
If yes, this is something to solve in the onChange lambda above, not in the visual transformation. But we should make it impossible to add more than 12 anyway. Unless it already is this way of course and I missed it here.

Reply:
Yep, didn't check the state, thanks! Fixed it now.